pyrometheus/codegen/python.py

```diff
@@ -244,10 +244,10 @@ def _pyro_norm(self, argument, normord):
             return np.abs(argument)
         return self.usr_np.linalg.norm(argument, normord)
 
-    def species_name(self, species_index):
+    def get_species_name(self, species_index):
```

Comment:
I'm on the fence about `get_` here. The method takes an argument, so it is not returning a property. Thoughts @henryleberre ?